### ISSUE TITLE
Accessibility of honeypot field

### ISSUE THREAD

**Issue by eddogan** [wpninjas/ninja-forms#3660]
Honeypot field is rendering the input field within label tag, and getting an empty label error, failing this criteria: 

https://www.w3.org/TR/WCAG20-TECHS/F68
